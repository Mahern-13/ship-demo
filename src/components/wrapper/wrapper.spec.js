import React from "react";
import { shallow } from "enzyme";
import Wrapper from "./index";

describe("Wrapper", () => {
  it("renders the component", () => {
    const wrapper = shallow(<Wrapper />);
    expect(wrapper).toMatchSnapshot();
  });

  it("renders after custom styling", () => {
    const wrapper = shallow(<Wrapper styling={{ padding: "0" }} />);
    expect(wrapper).toMatchSnapshot();
  });
});
