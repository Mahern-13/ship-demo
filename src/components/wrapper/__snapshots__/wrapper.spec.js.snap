// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wrapper renders after custom styling 1`] = `ShallowWrapper {}`;

exports[`Wrapper renders the component 1`] = `ShallowWrapper {}`;
